### ISSUE TITLE
temptv

### ISSUE THREAD

**Issue by bobert100** [Tempest0580/Temptv#60]
USA/TV and others on temptv haven't been working for weeks now. Some channels work, most don't What the heck is the problem?  I have tried and tried. I have reinstalled it but to no avail
        Bob

**Comment by joe71101** [comment]
Ustvgo wants everybody to purchase nord vpn.  Therefore all channels on ustvgo require nord vpn.  To work around using nord and use your current vpn, simply connect your VPN to a European country such as Denmark and channels will work again

**Comment by simakuutio** [comment]
Nope, any vpn works (or proper personal location in certain countries). I
listed those countries I found in another thread. Quite many european
country works just fine...
